adapter/src/main/kotlin/de/fraunhofer/iem/spha/adapter/tools/osv/OsvAdapter.kt

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright (c) 2024 Fraunhofer IEM. All rights reserved.
+ *
+ * Licensed under the MIT license. See LICENSE file in the project root for details.
+ *
+ * SPDX-License-Identifier: MIT
+ * License-Filename: LICENSE
+ */
+
+package de.fraunhofer.iem.spha.adapter.tools.osv
+
+import de.fraunhofer.iem.spha.adapter.AdapterResult
+import de.fraunhofer.iem.spha.adapter.kpis.cve.CveAdapter
+import de.fraunhofer.iem.spha.model.adapter.osv.OsvScannerDto
+import de.fraunhofer.iem.spha.model.adapter.vulnerability.VulnerabilityDto
+import java.io.InputStream
+import kotlinx.serialization.ExperimentalSerializationApi
+import kotlinx.serialization.json.Json
+import kotlinx.serialization.json.decodeFromStream
+
+object OsvAdapter {
+    private val jsonParser = Json {
```

Comment:
Is this something we can make global for the library, or should this be adapter specific?

Reply:
it might become adapter specific. 
However, as long as the config is the same we could use a global object for that I guess. I'll consider this in a dedicated PR